hw5/task_3.py

```diff
@@ -0,0 +1,17 @@
+# Задача № 3 домашнего задания 5. Генератор чисел Фибоначчи.
+
+def fibonacci_generator():
+    yield 0
+    a = 1
+    b = 1
```

Comment:
Почему бы не сделать так
```suggestion
    a, b = 0, 1
```
Ноль вернётся уже в цикле.